operator/pkg/controllers/master/podmonitors.go

```diff
@@ -82,3 +86,35 @@ func etcdPodMonitorFor(controlPlane *v1alpha1.ControlPlane) monitoringv1.PodMoni
 		PodMetricsEndpoints: []monitoringv1.PodMetricsEndpoint{{Port: "metrics"}},
 	}
 }
+
+func kcmPodMonitorFor(controlPlane *v1alpha1.ControlPlane) monitoringv1.PodMonitorSpec {
+	return monitoringv1.PodMonitorSpec{
+		JobLabel:            fmt.Sprintf("%s-controller-manager", controlPlane.ClusterName()),
+		NamespaceSelector:   monitoringv1.NamespaceSelector{MatchNames: []string{controlPlane.Namespace}},
+		Selector:            metav1.LabelSelector{MatchLabels: map[string]string{object.AppNameLabelKey: "kube-controller-manager"}},
+		PodMetricsEndpoints: []monitoringv1.PodMetricsEndpoint{{Port: "metrics"}},
+	}
+}
+
+func schedulerPodMonitorFor(controlPlane *v1alpha1.ControlPlane) monitoringv1.PodMonitorSpec {
+	return monitoringv1.PodMonitorSpec{
+		JobLabel:            fmt.Sprintf("%s-scheduler", controlPlane.ClusterName()),
+		NamespaceSelector:   monitoringv1.NamespaceSelector{MatchNames: []string{controlPlane.Namespace}},
+		Selector:            metav1.LabelSelector{MatchLabels: map[string]string{object.AppNameLabelKey: "kube-scheduler"}},
+		PodMetricsEndpoints: []monitoringv1.PodMetricsEndpoint{{Port: "metrics"}},
+	}
+}
+
+func authenticatorPodMonitorFor(controlPlane *v1alpha1.ControlPlane) monitoringv1.PodMonitorSpec {
+	return monitoringv1.PodMonitorSpec{
+		JobLabel:          fmt.Sprintf("%s-authenticator", controlPlane.ClusterName()),
+		NamespaceSelector: monitoringv1.NamespaceSelector{MatchNames: []string{controlPlane.Namespace}},
+		Selector:          metav1.LabelSelector{MatchLabels: iamauthenticator.Labels},
+		PodMetricsEndpoints: []monitoringv1.PodMetricsEndpoint{{
+			Port: "metrics", Scheme: "https",
+			TLSConfig: &monitoringv1.PodMetricsEndpointTLSConfig{
+				SafeTLSConfig: monitoringv1.SafeTLSConfig{InsecureSkipVerify: true},
```

Comment:
Is it possible to use 
`SafeTLSConfig: monitoringv1.SafeTLSConfig{CA: IAMAUTHENTICATOR_CA}`?